Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Simple Fedora based Docker container for running Boxes.py
+
+# Build the docker container
+# docker build -t boxes.py .
+
+# Run the docker container
+# docker run -ti -p 4000:8000 boxes.py
+
+# Run the docker container with a bind mount (Windows Command Line)
```

Comment:
This looks weird. Why would you only bind mount the scripts directory? The idea for bind mounting is to use the copy of boxes.py outside the container - not just the script. That way any changes to the work dir are visible without rebuilding the container.

Reply:
The problem of bind the full directory structure is this error

> RuntimeError: dictionary changed size during iteration

```
C:\Users\user1\Documents\GitHub\boxes>docker run -ti -p 4000:8000 -v %cd%:/boxes boxes.py
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python3.10/threading.py", line 1009, in _bootstrap_inner
    self.run()
  File "/boxes/scripts/boxesserver", line 75, in run
    if not self.filesOK():
  File "/boxes/scripts/boxesserver", line 64, in filesOK
    self._addModules()
  File "/boxes/scripts/boxesserver", line 55, in _addModules
    for name, module in sys.modules.items():
RuntimeError: dictionary changed size during iteration
BoxesServer serving on host:port :8000...
```